server/core/src/main/java/dev/slimevr/tracking/trackers/udp/FeatureFlags.kt

```diff
@@ -43,6 +43,7 @@ class FirmwareFeatures {
 enum class ServerFeatureFlags {
 	/** Server can parse bundle packets: `PACKET_BUNDLE` = 100 (0x64). */
 	PROTOCOL_BUNDLE_SUPPORT,
+	PROTOCOL_BUNDLE_V2_SUPPORT,
```

Comment:
Overall LGTM, but could use some docs, something like this would be more descriptive I think:
```suggestion
	/** Server can parse bundle packets with compact headers and packed IMU rotation/acceleration frames:
		- `PACKET_BUNDLE_COMPACT` = 101 (0x65),
		- `PACKET_ROTATION_AND_ACCELERATION` = 23 (0x17). */
	PROTOCOL_BUNDLE_COMPACT_SUPPORT,
```

Reply:
@0forks done, thank you